### PR TITLE
Experimental

### PR DESCRIPTION
**Pull request by Yuerino**
Update optimizer

### CODE REVIEW

**Comment by uwuplant** [comment]
Update optimizer

File: optimizer.py [hunk 1]
```diff
@@ -1,496 +1,674 @@
...
+        #              'intelligence': 9, 'attack speed': 4},
+        #     'legendary': {'strength': 12, 'defense': 12, 'speed': 2, 'health': 12, 'crit chance': 12, 'crit damage': 12,
+        #                   'intelligence': 12, 'attack speed': 5},
+        #     'mythic': {'strength': 15, 'defense': 15, 'speed': 2, 'health': 15, 'crit chance': 15, 'crit damage': 15,
+        #                'intelligence': 15, 'attack speed': 6},
+        #     'blacksmith': False
+        # }
+    },
+    'talisman': {
+		'bizzare': {
```
Comment (uwuplant) [inline optimizer.py:322]:
```suggestion
		'bizzare': {
```
bizarre

Comment (Yuerino) [reply optimizer.py:322]: thank you blessed soul

**Comment by uwuplant** [comment]
pets.py:
following pets need `'ability': None` added: bat, ocelot, guardian, jellyfish
following pets need `'ability'` moved above `'type'`: black cat, baby yeti, dolphin

**Pull request closed without merging**